components/sections/Hero.tsx

```diff
@@ -2,7 +2,10 @@ import Image from 'next/image';
 
 const HeroSection = () => {
   return (
-    <section className='flex w-full flex-col justify-center gap-3 pt-5 md:flex-row md:justify-start md:pt-10'>
+    <section
+      data-cy='hero'
```

Comment:
It should be consistent follow either `className then data` / `data then className`